app/app.component.html

```diff
@@ -44,7 +45,7 @@
             (click)="viewData()"
             >Data View <i class="fa fa-external-link-square"></i
           ></a>
-        </span>
+        </span>-->
```

Comment:
Why these pieces of code are commented? Let's remove them if we don't need them. Otherwise, I would leave a comment

Reply:
This tool supposes to be used for building ES query. Let's hide these pieces for now because it is not clear if we will continue using it as part of RXB, or extract to somewhere else.